### ISSUE TITLE
Memership sites Forum post

### ISSUE THREAD

**Issue by Thorbjorg2** [Simplero/Roadmap#402]
I think it is difficult to se and be sure if my members have seen and read my post in forum. A "like button" thumbs up / would be helpful, as it is in the old space forums.  I often use  facebook groups as Forum sites for my members and there admin / me, can see how many have actually seen the post which is rather helpful. I guess im to used to that feature and miss it in Simplero. And the thumbs up button!

**Comment by JesperChristophersen** [comment]
Agreed - thumbs up would be very nice to have in the new forum, - and it would be extra great, if it could also show who gave a thumbs up.

**Comment by Ryan-Henderson** [comment]
Agreed - That seems like a feature that I would use as well. I will add a vote to that request.
Thanks, Ryan
